src/xenia/kernel/kernel_state.cc

```diff
@@ -329,21 +336,21 @@ object_ref<UserModule> KernelState::LoadUserModule(const char* raw_name) {
 void KernelState::TerminateTitle(bool from_guest_thread) {
   auto global_lock = global_critical_region_.Acquire();
```

Comment:
need to call into the xapps here to have them kill/reset their state (like playlists in the xmp app, etc)